subsys/mgmt/mcumgr/transport/src/smp.c

```diff
@@ -151,10 +162,21 @@ int smp_transport_init(struct smp_transport *smpt)
 
 	k_work_init(&smpt->work, smp_handle_reqs);
 	k_fifo_init(&smpt->fifo);
-
+#ifdef CONFIG_SMP_CLIENT
+	k_fifo_init(&smpt->tx_fifo);
+	/* Store Transport pointer */
+	smpt_transport = smpt;
```

Comment:
As mentioned before, this is not acceptable

Reply:
I mention add my comment that this is still open. I have fix coming to this soon